.tekton/collector-pull-request.yaml

```
@@ -221,12 +225,27 @@ spec:
       - name: basic-auth
         workspace: git-auth
 
+    - name: prepare-rhel-rpm-subscriptions
+      runAfter:
+      - clone-repository
+      workspaces:
+      - name: source
+        workspace: workspace
+      - name: subscription-manager-activation-key
+        workspace: subscription-manager-activation-key
+      taskSpec:
+        steps:
+        # TODO(ROX-20651): use content sets instead of subscription manager for access to RHEL RPMs once available.
+        - name: smuggle-activation-key
+          image: registry.access.redhat.com/ubi8/ubi:latest
+          script: exec /workspace/source/source/.konflux/scripts/subscription-manager-bro.sh smuggle
```

Don't know if this really matters, but you could also resolve workspace path dynamically

```suggestion
          script: exec "$(workspaces.source.path)/source/.konflux/scripts/subscription-manager-bro.sh" smuggle
```

Up to you.